Add Note to Install OpenLDAP

Received the following error when running `python setup.py develop`
```
    In file included from Modules/LDAPObject.c:9:0:
    Modules/errors.h:8:18: fatal error: lber.h: No such file or directory
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```

Should save a few minutes by specifying it's a prerequisite.  Maybe provide platform specific instruction?  `apt-get install libldap2-dev` ?

--- a/README.md
+++ b/README.md
@@ -3,8 +3,11 @@ Oncall
 
 Initial setup
 -------------
+### Prerequisites
 
-Install dependencies:
+  * Debian/Ubuntu - `apt-get install libldap2-dev`
+
+### Install
 
 ```
 python setup.py develop

